src/Main.cpp

```diff
@@ -99,6 +103,9 @@ int main(int argc, char* argv[]) {
 
   seissol::MPI::mpi.init(argc, argv);
   const int rank = seissol::MPI::mpi.rank();
+#ifdef FP_CHECK
```

Comment:
Would it cause any problem to make it an environment variable instead?
(that's usually more convenient to handle than a CMake flag)

Reply:
Done.